src/store/modules/job/actions.ts

```diff
@@ -751,6 +751,35 @@ const actions: ActionTree<JobState, RootState> = {
     } catch (err) {
       console.error(err);
     }
+  },
+  async fetchMoreJobs({ commit }, payload) {
+    const params = {
+      "inputFields": {
+        ...payload.inputFields
+      } as any,
+      "fieldList": [ "systemJobEnumId", "runTime", "tempExprId", "parentJobId", "serviceName", "jobId", "jobName", "currentRetryCount", "statusId", "productStoreId", "runtimeDataId", "enumName", "shopId", "description", "enumTypeId" ],
+      "noConditionFind": "Y",
+      "viewSize": process.env.VUE_APP_VIEW_SIZE,
```

Comment:
Pass view size in payload instead of using the env variable directly here.

Reply:
sure sir